src/App.js

```diff
@@ -1,17 +1,15 @@
-import './App.css';
 import React from 'react';
-import { Provider } from 'react-redux'; // Import the Provider component
+import { Provider } from 'react-redux';
 import { Route, Routes } from 'react-router-dom';
-import store from './redux/store'; // Import your Redux store
+import './App.css';
+import Nav from './components/Nav';
+import store from './redux/store';
 import Books from './components/Books';
 import Categories from './components/Categories';
-import Nav from './components/Nav';
 
 function App() {
   return (
     <Provider store={store}>
```

Comment:
-  You have done great so far but there are few changes you need to make before moving to the next milestone. :+1: :+1: 

- [ ] Looking at the screenshot below, from `Book.js`, it seems you also wrap the component with the redux provider, that is not needed as you already did that in the App.js file. I'll suggest you remove that. :+1: :+1: 

- - Screenshot for reference:
 
![Screenshot from 2023-08-28 11-14-40](https://github.com/Kaiserabbas/bookstore/assets/107632780/ee0d77cb-6e5f-446f-b12e-f3f3464a9e65)


- [ ] Also following the best convention, I'll suggest you remove the `<Provider>` from App.js and move it to the index.js file. Like this:

```sh
import React from 'react';
import * as ReactDOM from 'react-dom/client';
import { BrowserRouter } from 'react-router-dom';
import App from './App';
import reportWebVitals from './reportWebVitals';

// Redux
import { Provider } from 'react-redux';
import store from './redux/store';

const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <BrowserRouter>
      <Provider store={store}>
        <App />
      </Provider>
    </BrowserRouter>
  </React.StrictMode>
);

// If you want to start measuring performance in your app, pass a function
// to log results (for example: reportWebVitals(console.log))
// or send to an analytics endpoint. Learn more: https://bit.ly/CRA-vitals
reportWebVitals();

```